proof_of_concept/policy_replication.py

```diff
@@ -84,7 +84,7 @@ def _update(self) -> None:
         # removed ones that disappeared
         removed_servers = [
                 server for server in self._other_stores
-                if server not in list(zip(*new_servers))[1]]
+                if server not in list(zip(*new_servers))[1]]  # type: ignore
```

Comment:
Mypy is complaining here with: "error: Unsupported right operand type for in ("object")". From a small inspection this code does not seem wrong.

Reply:
i.e. the result of `list(zip(*new_servers))[1]` is a tuple

Reply:
Agreed, this looks like a mypy issue, it has these things from time to time.